.travis.yml

```diff
@@ -27,6 +27,9 @@ before_script:
   - rustup toolchain install nightly-2020-04-23
   - rustup default nightly-2020-04-23
   - rustup component add rustfmt
+  - ulimit -n 8192
+  - pushd /tmp && wget https://github.com/aws/aws-sdk-cpp/archive/1.7.111.tar.gz -O /tmp/aws-sdk.tar.gz && tar -xvf /tmp/aws-sdk.tar.gz > /dev/null && popd
```

Comment:
cool. looks like the newest sdk is 1.8.9? https://github.com/aws/aws-sdk-cpp/releases